### ISSUE TITLE
[Bug]: Chrome stucked with new tab page when connect_over_cdp

### ISSUE THREAD

**Issue by tdRPA** [microsoft/playwright-python#2488]
### Version

1.45.0

### Steps to reproduce

1. open Chrome manully by `"C:\Program Files\Google\Chrome\Application\chrome.exe" --remote-debugging-port=9222`
2. go to "bing.com" site with the default one tab
3. connect Chrome 
```
from playwright.sync_api import sync_playwright
play=sync_playwright()
playDriver=play.start()

if __name__ == '__main__':
    CDPPort='9222'
    currentBrowser=playDriver.chromium.connect_over_cdp('http://127.0.0.1:%s' % CDPPort,timeout=30000)
    
    print('connected')
    
    import time
    time.sleep(500000)
```
4. back to Chrome, open new tab and go to "bing.com" site
5. open another new tab and go to "bing.com" site, then this tab page is stucked
6. terminate python process in step 3, the stucked page became OK

### Expected behavior

playwright not make Chrome sutcked

### Actual behavior

playwright make Chrome sutcked

### Environment

```Text
- Operating System: windows server 2016
- CPU: x64
- Browser: Chrome 126.0.6478.182 64bit 
- Python Version: 3.8.10
- Other info:
```

**Comment by mxschmitt** [comment]
See here: https://playwright.dev/python/docs/library#timesleep-leads-to-outdated-state

TL;dr: never use `time.sleep` and use `page.wait_for_timeout` or `[asyncio.sleep()](https://stackoverflow.com/questions/56729764/asyncio-sleep-vs-time-sleep)` isntead.
